aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie/Models/MovieNoEF.cs

```diff
@@ -0,0 +1,19 @@
+﻿#define First
+#if First
+
+#region MovieNoEF
```

Comment:
I recommend prefixing this region name with "snippet_" to make it obvious that it's used by a snippet import in the doc.